engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -305,6 +305,8 @@ class AlertReceiveChannel(IntegrationOptionsMixin, MaintainableObject):
     alert_group_labels_template: str | None = models.TextField(null=True, default=None)
     """Stores a Jinja2 template for "advanced label templating" for alert group labels."""
 
+    additional_settings: typing.Optional[typing.Dict] = models.JSONField(null=True, default=None)
```

Comment:
_nit_, maybe just a preference thing:
```suggestion
    additional_settings: dict | None = models.JSONField(null=True, default=None)
```